code/game/objects/structures/mineral_doors.dm

```diff
@@ -2,27 +2,27 @@
 //machineryness
 
 /obj/structure/mineral_door
-	name = "mineral door"
+	name = "metal door"
 	density = 1
 	anchored = 1
 	opacity = 1
 
 	icon = 'icons/obj/doors/mineral_doors.dmi'
 	icon_state = "metal"
 
-	var/mineralType = "metal"
+	var/initial_state
 	var/state = 0 //closed, 1 == open
 	var/isSwitchingStates = 0
 	var/close_delay = -1 //-1 if does not auto close.
 	var/hardness = 1
-	var/oreAmount = 7
+	var/sheetType = /obj/item/stack/sheet/metal
+	var/sheetAmount = 7
 	var/openSound = 'sound/effects/stonedoor_openclose.ogg'
 	var/closeSound = 'sound/effects/stonedoor_openclose.ogg'
 
 /obj/structure/mineral_door/New(location)
 	..()
-	icon_state = mineralType
-	name = "[mineralType] door"
+	initial_state = icon_state

```

Comment:
I'm leaving the ability for admins to maybe edit the door's icon states(because maybe they want to do that?)